packages/browser/src/components/explorer/FileExplorerProvider.tsx

```diff
@@ -39,7 +40,7 @@ export default function FileExplorerProvider({ rootPath }: Props) {
 			setPath(entry.path)
 		} else {
 			try {
-				const entity = await getBook(entry.path)
+				const entity = await getBook(entry.path, sdk)
```

Comment:
I wonder why the lints didn't catch this. I'll have to revisit the frontend lint scripts

Reply:
That's a good point, I found the problem easily because ESLint threw an error in VS Code and underlined it in red for me.

Reply:
Well actually here's your answer: https://github.com/typescript-eslint/typescript-eslint/issues/352#issuecomment-472597115

The salient part:
> we purposely do not care about errors that should arise as a result of type checking errors.
We only care that the code is syntactically valid (i.e. can be parsed without errors), not about whether it's semantically valid (i.e. don't care if it is type sound).

We need to use `tsc` to catch this sort of thing.

Reply:
Ahh yeah, okay. That makes sense. I could have sworn I defined a tsc script somewhere to check types but I'll take a look tomorrow